extension/src/background.js

```diff
@@ -833,6 +839,9 @@ chrome.runtime.onMessage.addListener((info: any, sender: chrome$MessageSender) =
         lwarn("Suppressing special background page %s", aurl);
         return;
     }
+
+    console.info("Registering background page callbacks in tab %s", aurl);
```

Comment:
hmm, why you moved it lower?

Reply:
because if any of the anti-conditions are true, you already get a message indicating what's happening and /why it will not call `initBackground()`. Admittedly, some of them only in a more verbose logging level. I.e.:
`console.debug("ignoring %o %o; %s", info, sender, backgroundInitialised);`
`console.debug("background already initialised");`
`lwarn("Suppressing special background page %s", aurl);`

So, "Registering background page callbacks" is both a misleading information and unnecessary noise. We should probably move the statement all the way into `initBackground()`?

A more humane response: i was just shushing the logging a bit to make it easier to orient myself:)

Reply:
Ah ok, makes sense!